### PR TITLE
Fix __str__ for Post and BlogCategory w/ no translation

### PR DESCRIPTION
**Pull request by sidneijp**
Fix string representation for `Post`s and `BlogCategory`s without translation instance related.

### CODE REVIEW

File: djangocms_blog/models.py [hunk 1]
```diff
@@ -164,7 +164,8 @@ def get_absolute_url(self, lang=None):
         )
 
     def __str__(self):
-        return self.safe_translation_getter('name', any_language=True)
+        default = ugettext('BlogCategory (no translation)')
```
Comment (yakky) [inline djangocms_blog/models.py]: Why not ugettext_lazy?

Comment (sidneijp) [reply djangocms_blog/models.py]: Cause with the lazy function it's raising:

`TypeError: __str__ returned non-string (type __proxy__)`

Comment (yakky) [reply djangocms_blog/models.py]: I see :+1:

**Comment by yakky** [comment]
Fix #443

**Comment by yakky** [comment]
Thansk @rafaelgontijo @sidneijp  for your efforts in tackling this!

**Pull request merged**